components/collator/src/elements.rs

```diff
@@ -154,9 +153,14 @@ pub(crate) const EMPTY_U16: &ZeroSlice<u16> =
 const SINGLE_U16: &ZeroSlice<u16> =
     ZeroSlice::<u16>::from_ule_slice(&<u16 as AsULE>::ULE::from_array([0xFFFD]));
 
-const SINGLE_U24_ARR: [u8; 3] = [0xFD, 0xFF, 00];
-const SINGLE_U24_SLICE: &[U24] = &[U24(SINGLE_U24_ARR)];
-const SINGLE_U24: &ZeroSlice<U24> = unsafe { core::mem::transmute(SINGLE_U24_SLICE) };
+/// Slice that we need to transmute
+const EMPTY_CHAR_SLICE: &[char] = &[];
+/// An empty `&ZeroSlice<char>`
+pub const EMPTY_CHAR: &ZeroSlice<char> = unsafe { core::mem::transmute(EMPTY_CHAR_SLICE) };
+
+const SINGLE_CHAR_ARR: [u8; 3] = [0xFD, 0xFF, 00];
+const SINGLE_CHAR_ULE: CharULE = unsafe { core::mem::transmute(SINGLE_CHAR_ARR) };
+const SINGLE_CHAR: &ZeroSlice<char> = ZeroSlice::<char>::from_ule_slice(&[SINGLE_CHAR_ULE]);
```

Comment:
Can you find a better name for this? What char is it?

Reply:
Renamed.